packages/solvuu_build/solvuu_build.0.0.1/descr

```diff
@@ -0,0 +1 @@
+Solvuu's build system.
```

Comment:
Why not.  Since your readme is markdown, you could use comments to delimit a (small) section to be copied to the `descr` file.